next/components/forms/steps/Summary/StepButtonGroup.tsx

```diff
@@ -10,7 +11,7 @@ interface StepButtonGroupProps {
   previous: () => void
   skip: () => void
   submitStep: () => void
-  submitForm: () => void
+  submitForm: (e: PressEvent) => void
```

Comment:
are you using the press event ? because if not and it's there just to match the expected typescript type, would rather change wherever this is used from `onPress={submitForm}` to `onPress={() => submitForm()}` then change this type

but if it's unavoidable of if the event is actually used then you can keep it